tools/sof-kernel-log-check.sh

```diff
@@ -328,6 +318,15 @@ ignore_str="$ignore_str"'|asix .*: Failed to enable software MII access'
 # e1000e 0000:00:1f.6 enp0s31f6: Hardware Error
 ignore_str="$ignore_str"'|e1000e .*: Hardware Error'
 
+# iwlwifi net adaptor logs can be ignored
+# origin logs seen on CML platforms
+# iwlwifi 0000:00:14.3: Direct firmware load for iwlwifi-QuZ-a0-hr-b0-56.ucode failed with error -2
+# iwlwifi 0000:00:14.3: Direct firmware load for iwl-debug-yoyo.bin failed with error -2
+ignore_str="$ignore_str"'|iwlwifi .*: Direct firmware load for .* failed with error -2'
```

Comment:
For this kind of scenario, I suggest to use `.+`, because `*` matches to everything, and may not proper to use here. 
just
```
ignore_str="$ignore_str"'|iwlwifi .+'
```

Reply:
Yep, I think we can ignore all iwlwifi related error logs from our discussion. and `.+` would be better.
Actually `.*` and `.+` will both match everything, but `.*` will match even when there is nothing while `.+` need at least one char.

Reply:
The problem here is our ignore_str will look like `iwifi: .*|some-other-ignore-str`,  I am not sure if the `*` will eat the `|`, which used to connect two ignore_strs.

Reply:
The `|` is not really "connecting" anything, it's a logical OR part of the regular expression.
The `*` affects only the previous expression.

See regex suggestion in the main thread.